llpc/translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -6445,6 +6384,19 @@ bool SPIRVToLLVM::translate(ExecutionModel entryExecModel, const char *entryName
     }
   }
 
+  if (scratchBoundsChecksEnabled()) {
+    // Insert the scratch out of bounds checks for any feasible memory instruction. The SPIRV to LLVM memop mapping
+    // gets filled while translating OpLoads and OpStores.
+    for (Function &f : m_m->getFunctionList()) {
+      Function *ff = &f;
```

Comment:
I think we can avoid ff as local variable, just pass &func as the function argument.

Reply:
Are you sure? ```ff``` gets used three times.